src/LSST/control/ControlMain.h

```diff
@@ -47,20 +56,52 @@ class ControlMain {
     /// Verify threads are stopped and joined.
     ~ControlMain();
 
-    /// Return a reference to the global ControlMain instance.
-    /// @throws `ConfigException` if `setup` has not already been called.
-    static ControlMain& get();
-
     /// Return a shared pointer to the global ControlMain instance.
     /// @throws `ConfigException` if `setup` has not already been called.
     static Ptr getPtr();
 
+    /// &&& doc
+    void run(int argc, const char* argv[]);
+
+    /// Returns true once the server is running.
+    bool getRunning() { return _running; }
+
+    /// Return a pointer to `_simCore`.
+    std::shared_ptr<simulator::SimCore> getSimCore() { return _simCore; }
```

Comment:
What's the reason for this method? I've not found any uses of the method in the branch.

Reply:
The simulator is only used in testing. Allowing access makes tests more meaningful. Say for the motor power on test, the client thread (which is the main thread) can send the message to turn on the motor power. Then it can get a pointer to the simulator to verify the motor power is really on, as opposed to just believing what the server tells it.